content/ui-patterns/common-actions.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: Common actions
+description: Actions that are commonly used across various workflows across the application.
```

Comment:
Do we need this page if we don't have any content for it?

At the very least, we should show that `Note` we show for the components.

Reply:
I...am baffled where this came from. deleting